woocommerce/payment-gateway/External_Checkout/Admin.php

```diff
@@ -429,14 +424,13 @@ protected function should_display_shipping_based_tax_notice() {
 	 *
 	 * @return string
 	 */
-	protected function get_shipping_based_tax_notice() {
+	protected function get_shipping_based_tax_notice() : string {
 
 		return sprintf(
-			/** translators: Placeholders: %1$s - external checkout label, %2$s - <a> tag, %3$s - </a> tag, %4$s - <strong> tag, %5$s - </strong> tag */
-			__( '%4$s%1$s Notice!%5$s Your store %2$scalculates taxes%3$s based on the shipping address, but %1$s %4$sdoes not%5$s share customer shipping information with your store for orders with only virtual products. These orders will have their taxes calculated based on the shop address instead.', 'woocommerce-plugin-framework' ),
+			/* translators: Placeholders: %1$s - Checkout method label, %2$s - Opening HTML <a> tag, %3$s - Closing HTML </a> tag, %4$s - Opening HTML <strong> tag, %5$s - closing HTML </strong> tag */
+			__( 'Your store %1$scalculates taxes%2$s based on the shipping address, but %1$s %4$sdoes not%5$s share customer shipping information with your store for orders with only virtual products. These orders will have their taxes calculated based on the shop address instead.', 'woocommerce-plugin-framework' ),
```

Comment:
I changed slightly this notice as it was hard to parse by the translator - basically I just trimmed the "Notice!" bit which I too found a bit weirdly worded